src/Okta.Sdk/IUsersClient.cs

```diff
@@ -22,6 +22,15 @@ public partial interface IUsersClient : IAsyncEnumerable<IUser>
         /// <returns>The created user.</returns>
         Task<IUser> CreateUserAsync(CreateUserWithoutCredentialsOptions options, CancellationToken cancellationToken = default(CancellationToken));
 
+        /// <summary>
+        /// Creates a new user in your Okta organization a user with a specified hashed password.
+        /// </summary>
+        /// <remarks>See <a href="https://developer.okta.com/docs/reference/api/users/#create-user-with-imported-hashed-password">Create User with Imported Hashed Password</a> in the documentation.</remarks>
```

Comment:
I would remove " in the documentation" here.

Reply:
It seems this wasn't resolved yet. @andriizhegurov-okta can you please remove "in the documentation" from the remarks? Thanks!